source/collections/PointerVector.ooc

```diff
@@ -72,43 +75,31 @@ PointerVector: abstract class {
 		else
 			memcpy(destination, source, length)
 	}
-
 	operator [] (index: Int) -> Pointer {
 		this _backend[index]
 	}
-
 	operator []= (index: Int, item: Pointer) {
 		this _backend[index] = item
 	}
 }
 
-
 PointerHeapVector: class extends PointerVector {
-	init: func(count: Int) {
+	init: func (count: Int) {
 		super(count)
 	}
-
-	_allocate: func(count: Int) {
+	_allocate: func (count: Int) {
 		this _backend = gc_realloc(this _backend, count * Pointer size)
 	}
-
-	__destroy__: func {
-		super()
-		gc_free(this _backend)
-	}
 }
 
 PointerStackVector: class extends PointerVector {
-	init: func(data: Pointer*, count: Int) {
+	init: func (data: Pointer*, count: Int) {
 		super(data, count)
 	}
-
-	// TODO: Why does this function exist here?
-	_allocate: func(count: Int)
-
-	resize: override func(count: Int) {
+	_allocate: func (count: Int)
```

Comment:
I'm not merging this. Remove this function here and in the base class, since it's not used in either place. Keep the function in `PointerHeapVector` where it's actually used.